src/app/pages/LiveRadioPage/__snapshots__/index.test.tsx.snap

```diff
@@ -1075,7 +1075,6 @@ exports[`Radio Page Main should match snapshot for Canonical 1`] = `
 }
 
 <div>
-  <noscript />
```

Comment:
Oh interesting... I wonder why this is different? Does the test data for the snapshot also need to be updated?

Reply:
This is a weird one, because the empty noscript doesn't do anything, so I'm not sure why it was there in the first place?

Reply:
It must be from one of the components that used to be on the page, but isn't anymore... What if it was the ATIAnalytics component, which is no longer appearing in the snapshot. Or does it really matter, as long as the real page renders the correct components...?

Reply:
I don't think we need to worry too much, the page itself + Analytics are working as expected so I think we can leave this as is.